File: programmers/숫자의 표현/answer.py
def solution(n):
    answer = 0
    
    for standard in range(1, n + 1):
        sum_val = 0
        
        for i in range(standard, n + 1):
            sum_val += i
            if sum_val == n:
                answer += 1
                break
            if sum_val > n:
                break
    
    return answer

File: programmers/피보나치 수/answer.java
class Solution {
    public int solution(int n) {
        int answer = 0;
        int fibMinusOne = 1;
        int fibMinusTwo = 0;

        for (int i = 2; i <= n; i++) {
            answer = (fibMinusOne + fibMinusTwo) % 1234567;
            fibMinusTwo = fibMinusOne % 1234567;
            fibMinusOne = answer;
        }

        return answer;
    }
}